### ISSUE TITLE
Add the Accessibility features that were added into Bootstrap

### ISSUE THREAD

**Issue by jgodi** [gwtbootstrap3/gwtbootstrap3#254]
Bootstrap v3.3.0 added a lot of accessibility features into their library. They should be ported over and added into GwtBootstrap3.

This will be a major change since we basically have to go back through a lot of the widgets and add these.

**Comment by coderwurst** [comment]
is this feature still a possibility or is it too complicated/ too big of a job?

**Comment by crevete** [comment]
It needs a lot of jobs. Please feel free to make some PR for this, thanks you
